src/ui/store/reducers/asideHeader.ts

```diff
@@ -46,7 +46,7 @@ export function getLandingNavigationSettings() {
 
 const initialSettings: AsideHeaderSettings = {
     withHelpCenter: true,
-    withNavigation: true,
+    withNavigation: !DL.USER.isFederationUserOnCommunity,
```

Comment:
We need to use more common naming here:
`!DL.USER.isFederationUserOnCommunity` -> `DL.withNavigation` for example

Reply:
added DL.HIDE_NAVIGATION